Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/NumberBoxSamplePage.xaml

```diff
@@ -24,11 +24,15 @@
 
 						<smtx:XamlDisplay UniqueKey="NumberBoxSamplePage_Fluent_Default"
 										  smtx:XamlDisplayExtensions.Header="A simple NumberBox">
-							<muxc:NumberBox Header="Enter a number:"
-											Minimum="10"
-											Maximum="100"
-											PlaceholderText="10"
-											Description="This is a description which is shown below the NumberBox"/>
```

Comment:
Would this hide a default style issue? Maybe the original style should be update from WinUI? /cc @kazo0

Reply:
This issue is the same in Windows, Description does not wrap at all (https://github.com/microsoft/microsoft-ui-xaml/blob/4873563b7953e2431caed4efcdc3b8261d1405cd/dev/NumberBox/NumberBox.xaml#L279-L284)

So this workaround is needed for all platforms, including Windows

cc @jeromelaban